modules/claims_api/app/controllers/claims_api/v0/claims_controller.rb

```diff
@@ -22,25 +22,7 @@ def index
       end
 
       def show
-        if (pending_claim = ClaimsApi::AutoEstablishedClaim.pending?(params[:id]))
-          render json: pending_claim,
-                 serializer: ClaimsApi::AutoEstablishedClaimSerializer
-        else
-          begin
-            evss_claim_id = ClaimsApi::AutoEstablishedClaim.evss_id_by_token(params[:id]) || params[:id]
-            fetch_and_render_evss_claim(evss_claim_id)
-          rescue EVSS::ErrorMiddleware::EVSSError
-            render json: { errors: [{ detail: 'Claim not found' }] },
-                   status: :not_found
-          end
-        end
-      end
-
-      private
-
-      def fetch_and_render_evss_claim(id)
-        claim = claims_service.update_from_remote(id)
-        render json: claim, serializer: ClaimsApi::ClaimDetailSerializer
+        super
```

Comment:
this is super bizarro. i wonder why you need to do this

Reply:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v0/forms/disability_compensation_controller.rb#L52-L54 I ran into the same problem when I tried consolidating our validate end point, I'm not sure if it's just how vets-api is configured or maybe even how our module is configured but it'll simply fail and not recognize the method if I remove it completely from the controller